packages/ember-metal/lib/mixin.js

```diff
@@ -102,8 +102,14 @@ function giveDescriptorSuper(meta, key, property, values, descs, base) {
     superProperty = superDesc;
   }
 
+  // add a default `_super` to ensure proper wrapping if `_super` is used in `property`
   if (superProperty === undefined || !(superProperty instanceof ComputedProperty)) {
-    return property;
+    superProperty = { _getter() {} };
+
+    // only provide a super setter if the property has one already
+    if (property._setter) {
+      superProperty._setter = function() {};
```

Comment:
you should use the ROOT that wrap uses for this, so it is marked as not having super explicitly